---
layout: page
title: Thompson Lab Papers
---
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>

**Papers**

<span class="__dimensions_badge_embed__" data-doi="10.1093/insilicoplants/diab001" data-style="large_rectangle" style="float:right;"></span>
* Yang KW, Chapman S, Carpenter N, Hammer G, McLean G, Zheng B, Chen Y, Delp E, Masjedi A, Crawford M, Ebert D, Habib A, **Thompson AM**, Weil C, Tuinstra MR (2021) [Integrating Crop Growth Models with Remote Sensing for Predicting Biomass Yield of Sorghum](https://doi.org/10.1093/insilicoplants/diab001) *G3* doi: 10.1093/insilicoplants/diab001

<span class="__dimensions_badge_embed__" data-doi="10.1093/g3journal/jkaa050" data-style="large_rectangle" style="float:right;"></span>
* Rogers A et al. (2021) [The Importance of Dominance and Genotype-by-Environment Interactions on Grain Yield Variation in a Large-Scale Public Cooperative Maize Experiment](https://doi.org/10.1093/g3journal/jkaa050) *G3* doi: 10.1093/g3journal/jkaa050

<span class="__dimensions_badge_embed__" data-doi="10.1002/aps3.11386" data-style="large_rectangle" style="float:right;"></span>
* Raju SKK, **Thompson AM**, Schnable JC (2020) [Advances in plant phenomics: From data and algorithms to biological insights](https://doi.org/10.1002/aps3.11386) *Applications in Plant Sciences* doi: 10.1002/aps3.11386

<span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-019-12646-z" data-style="large_rectangle" style="float:right;"></span>
* do Vale Martins L, Yu F, Zhao H, Dennison T, Lauter N, Wang H, Deng Z, **Thompson AM**, Semrau K, Rouillard JM, Birchler JA, Jiang J (2019) [Meiotic crossovers characterized by haplotype-specific chromosome painting in maize](https://doi.org/10.1038/s41467-019-12646-z) *Nature Communications* doi: 10.1038/s41467-019-12646-z

<span class="__dimensions_badge_embed__" data-doi="10.1016/j.plantsci.2018.06.018" data-style="large_rectangle" style="float:right;"></span>
* van Eeuwijk FA, Bustos-Korts D, Millet EJ, Boer MP, Kruijer W, **Thompson AM**, Malosetti M, Iwata H, Quiroz R, Kuppe C, Muller O, Blazakis KN, Yu K, Tardieu F, Chapman SC (2019) [Modelling strategies for assessing and increasing the effectiveness of new phenotyping techniques in plant breeding](https://doi.org/10.1016/j.plantsci.2018.06.018) *Plant Science* doi: 10.1016/j.plantsci.2018.06.018

<span class="__dimensions_badge_embed__" data-doi="10.3389/fdata.2019.00027" data-style="large_rectangle" style="float:right;"></span>
* Yu M, Ramamurthy KN, **Thompson AM**, Lozano AC (2019) [Simultaneous parameter learning and bi-clustering for multi-response models](https://doi.org/10.3389/fdata.2019.00027) *Frontiers in Big Data* doi: 10.3389/fdata.2019.00027

<span class="__dimensions_badge_embed__" data-doi="10.2135/cropsci2016.03.0188" data-style="large_rectangle" style="float:right;"></span>
* Massafaro M, **Thompson AM**, Tuinstra M, Dilkes B, Weil CF (2016) [Mapping the increased protein digestibility trait in the high-lysine sorghum mutant P721Q](https://doi.org/10.2135/cropsci2016.03.0188) *Crop Science* doi: 10.2135/cropsci2016.03.0188

<span class="__dimensions_badge_embed__" data-doi="10.3835/plantgenome2016.03.0028" data-style="large_rectangle" style="float:right;"></span>
* Bernardo R, **Thompson AM** (2016) [Germplasm architecture revealed through chromosomal effects for quantitative traits in maize](https://doi.org/10.3835/plantgenome2016.03.0028) *The Plant Genome* doi: 10.3835/plantgenome2016.03.0028

<span class="__dimensions_badge_embed__" data-doi="10.1007/s00138-015-0728-4" data-style="large_rectangle" style="float:right;"></span>
* Kelly D, Vatsa A, Mayham W, Ngo L, Thompson AM, Kazic T (2016) [An opinion on imaging challenges in phenotyping field crops](https://doi.org/10.1007/s00138-015-0728-4) *Machine Vision and Applications* doi: 10.1007/s00138-015-0728-4

<span class="__dimensions_badge_embed__" data-doi="10.1534/g3.115.017541" data-style="large_rectangle" style="float:right;"></span>
* **Thompson AM**, Schnable PS, Yu Jianming Timmermans MCP, Schnable PS, Crants JE, Scanlon MJ, Muehlbauer GJ (2015) [Diversity of maize shoot apical meristem architecture and its relationship to plant morphology](https://doi.org/10.1534/g3.115.017541) *G3: Genes, Genomes, Genetics* doi: 10.1534/g3.115.017541

<span class="__dimensions_badge_embed__" data-doi="10.1534/g3.114.011940" data-style="large_rectangle" style="float:right;"></span>
* **Thompson AM**, Crants J, Schnable PS, Yu J, Timmermans MCP, Springer NM, Scanlon MJ, Muehlbauer GJ (2014) [Genetic control of maize shoot apical meristem architecture](https://doi.org/10.1534/g3.114.011940) *G3: Genes, Genomes, Genetics* doi: 10.1534/g3.114.011940

<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pone.0030798" data-style="large_rectangle" style="float:right;"></span>
* Makarevitch I, **Thompson AM**, Muehlbauer GJ, Springer NM (2012) [Brd1 gene in maize encodes a brassinosteroid C-6 oxidase](https://doi.org/10.1371/journal.pone.0030798) *PLoS One* doi: 10.1371/journal.pone.0030798

<span class="__dimensions_badge_embed__" data-doi="10.1111/j.1365-313X.2008.03778.x" data-style="large_rectangle" style="float:right;"></span>
* Li L. Yu X, **Thompson AM**, Guo M, Yoshida S, Asami T, Chory J, Yin Y (2009) [Arabidopsis MYB30 is a direct target of BES1 and cooperates with BES1 to regulate brassinosteroid-induced gene expression](https://doi.org/10.1111/j.1365-313X.2008.03778.x) *The Plant Journal* doi: 10.1111/j.1365-313X.2008.03778.x

**Preprints and Conference Papers**

* Long L, Bassett A, Cichy K, **Thompson AM**, and Morris D. (2019) [Bean split ratio for dry bean canning quality and variety analysis](https://openaccess.thecvf.com/content_CVPRW_2019/papers/CVPPP/Long_Bean_Split_Ratio_for_Dry_Bean_Canning_Quality_and_Variety_CVPRW_2019_paper.pdf) *2019 IEEE Conference on Computer Vision and Pattern Recognition (CVPR) Workshops* 

<span class="__dimensions_badge_embed__" data-doi="10.1109/DSAA.2018.00052" data-style="large_rectangle" style="float:right;"></span>
* Olsen PA, Ramamurthy KN, Ribera J, Chen Y, **Thompson AM**, Luss R, Tuinstra M, Abe N (2018) [Detecting and counting panicles in sorghum images](https://doi.org/10.1109/DSAA.2018.00052) *2018 IEEE 5th International Conference on Data Science and Advanced Analytics* doi: 10.1109/DSAA.2018.00052

<span class="__dimensions_badge_embed__" data-doi="10.1109/IGARSS.2018.8519034" data-style="large_rectangle" style="float:right;"></span>
* Masjedi A, Zhao J, **Thompson AM**, Yang KW, Flatt JE, Crawford MM, Ebert DS, Tuinstra MR, Hammer G, Chapman S (2018) [Sorghum biomass prediction using UAV-based remote sensing data and crop model simulation](https://doi.org/10.1109/IGARSS.2018.8519034) *2018 IEEE International Geoscience and Remote Sensing Symposium* doi: 10.1109/IGARSS.2018.8519034

* Yu M, **Thompson AM**, Ramamurthy KN, Yang E, Lazano AC. [Multitask learning using task clustering with applications to predictive modeling and GWAS of plant varieties.](https://arxiv.org/abs/1710.01788) *arXiv* arXiv:1710.01788

* Tuinstra MR, Weil C, **Thompson AM**, Boomsma C, Crawford M, Habib A, Delp E, Cherkauer K, Biehl L, Abe N, Kshirsagar M, Lozano A, Ramamurthy KN, Olsen P, and Yang E. (2016) [Automated sorghum phenotyping and trait development platform](http://scholar.google.com/scholar_lookup?author=M.+R.+Tuinstra+&publication_year=2016&title=%E2%80%9CAutomated+sorghum+phenotyping+and+trait+development+platform,%E2%80%9D&journal=Proceedings+of+KDD+Workshop+on+Data+Science+for+Food,+Energy,+and+Water) *Proceedings of KDD Workshop on Data Science for Food, Energy, and Water (San Francisco, CA)*