---
layout: page
title: Schnable Lab Papers
---
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

**Preprints**

<div data-badge-type="2" data-doi="10.1101/310391" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* **Miao C**, **Yang J**, **Schnable JC** "[Optimizing the identification of causal variants across varying genetic architectures in crops.](https://doi.org/10.1101/310391)" <i>bioRxiv</i> doi: 10.1101/310391 

<div data-badge-type="2" data-doi="10.1101/187575" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* <b>Yan L</b>, <b>Lai X</b>, Rodriguez O, Mahboub S, Roston RL, <b>Schnable JC</b>. "<a href="https://doi.org/10.1101/187575">Parallels between artificial selection in temperate maize and natural selection in the cold-adapted crop-wild relative Tripsacum.</a>" <i>bioRxiv</i> doi: 10.1101/187575 



**2018**


<div data-badge-type="2" data-doi="10.1534/g3.118.200242" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* <b>Liang Z</b>, Gupta SK, Yeh CT, <b>Zhang Y</b>, <b>Ngu DW</b>, Kumar R, Patil HT, Mungra KD, Yadav DV, Rathore A, Srivastava RK, Gupkta R, <b>Yang J</b>, Varshney RK, Schnable PS, <b>Schnable JC</b> (2018) "[Phenotypic data from inbred parents can improve genomic prediction in pearl millet hybrids.](https://doi.org/10.1534/g3.118.200242)" <i>G3: Genes Genomes Genetics</i> doi: 10.1534/g3.118.200242
[Chinese Language Summary](https://mp.weixin.qq.com/s/8mGpB0Uel0omoC5QFGjKmA)

<div data-badge-type="2" data-doi="10.1038/s41598-018-28294-0" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* **Miao C**, Fang J, Li D, Liang P, Zhang X, **Yang J**, **Schnable JC**, Tang H (2018) "[Genotype-Corrector: improved genotype calls for genetic mapping.](https://doi.org/10.1038/s41598-018-28294-0)" <i>Scientific Reports</i> doi: 10.1038/s41598-018-28294-0

<div data-badge-type="2" data-doi="Pubs/Sruti_KDD_FEWS.pd" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* Alkhalifah N, Campbell DA, Falcon CM, ... **Schnable JC** (31 of 42 authors) ... Spalding EP, Edwards J, Lawrence-Dill CJ (2018) "Maize Genomes to Fields: 2014 and 2015 field season genotype, phenotype, environment, and inbred ear image datasets." <i>BMC Research Notes</i> (In Press)

<div data-badge-type="2" data-doi="10.2135/tppj2017.09.0007" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* Xu Y, Qiu Y, **Schnable JC**. (2018) "[Functional Modeling of Plant Growth Dynamics.](https://dl.sciencesocieties.org/publications/tppj/abstracts/1/1/170007)" <i>The Plant Phenome</i> doi: 10.2135/tppj2017.09.0007 <i>bioRxiv</i> doi: <a href="https://doi.org/10.1101/190967">10.1101/190967</a>
<i>An [invited webinar](https://www.youtube.com/watch?v=lw_WsWV6ox0) organized by the The Plant Phenome Journal where James Schnable describes the key advances from this paper for a wider audience.</i>

<div data-badge-type="2" data-doi="10.1177/1176934318764683" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* **Carvalho DS**, **Schnable JC**, Almeida AMR. (2018) "[Integrating phylogenetic and network approaches to study gene family evolution: the case of the AGAMOUS family of floral genes.](https://doi.org/10.1177/1176934318764683)" <i>Evolutionary Bioinformatics</i> doi: 10.1177/1176934318764683 <i>bioRxiv</i> doi: <a href="https://doi.org/10.1101/195669">10.1101/195669</a>

<div data-badge-type="2" data-doi="10.1016/j.molp.2017.12.010" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* <b>Liang Z</b>, <b>Schnable JC</b> (2018) "<a href="https://doi.org/10.1016/j.molp.2017.12.010">Functional Divergence Between Subgenomes and Gene Pairs After Whole Genome Duplications.</a>" <i>Molecular Plant</i> doi: 10.1016/j.molp.2017.12.010

<div data-badge-type="2" data-doi="10.1007/s11033-018-4144-5" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* Nani TF, <b>Schnable JC</b>, Washburn JD, Albert P, Pereira WA, Sobrinho FS, Birchler JA, Techia VH (2018) "<a href="https://doi.org/10.1007/s11033-018-4144-5">Location of low copy genes in chromosomes of Brachiaria spp.</a>" <i>Molecular Biology Reports</i> doi: 10.1007/s11033-018-4144-5

<div data-badge-type="2" data-doi="10.1111/tpj.13806" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* <b>Lai X</b>, <b>Yan L</b>, Lu Y, <b>Schnable JC</b>. (2018) "<a href="https://doi.org/10.1111/tpj.13806">Largely unlinked gene sets targeted by selection for domestication syndrome phenotypes in maize and sorghum.</a>" <i>The Plant Journal</i> doi: 10.1111/tpj.13806  <i>bioRxiv</i> doi: <a href="https://doi.org/10.1101/184424">10.1101/184424</a>


**2017**

<div data-badge-type="2" data-doi="10.1093/gigascience/gix117" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*  <b>Liang Z</b>, Pandey P, Stoerger V, Xu Y, Qiu Y, Ge Y, <b>Schnable JC</b>. (2017) "<a href="https://academic.oup.com/gigascience/advance-article/doi/10.1093/gigascience/gix117/4656251?guestAccessKey=71c1c32f-78fd-42c0-99a3-38e6c4fd8100">Conventional and hyperspectral time-series imaging of maize lines widely used in field trials.</a>" <i>GigaScience</i> doi: 10.1093/gigascience/gix117 <i>bioRxiv</i> doi: <a href="https://doi.org/10.1101/169045">10.1101/169045</a>  

<div data-badge-type="2" data-doi="10.1038/s41467-017-01450-2" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*  Gage J, Jarquin D, Romay M, ... <b>Schnable JC</b> (29th of 40 authors) .. Yu J, de Leon N. (2017) "<a href="https://doi.org/10.1038/s41467-017-01450-2">The effect of artificial selection on phenotypic plasticity in maize.</a>" <i>Nature Communications</i> doi: 10.1038/s41467-017-01450-2  

<div data-badge-type="2" data-doi="10.1038/s41598-017-13236-z" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*  Washburn JD, <b>Schnable JC</b>, Brutnell TP, Shao Y, <b>Zhang Y</b>, Ludwig M, Davidse G, Pires JC. (2017) "<a href="https://doi.org/10.1038/s41598-017-13236-z">Genome-Guided Phylo-Transcriptomics.</a>" <i>Scientific Reports</i> doi: 10.1038/s41598-017-13236-z  

<div data-badge-type="2" data-doi="10.1109/BIBM.2017.8217628" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*  Behera S, Deogun JS, <b>Lai X</b>, <b>Schnable JC</b>. (2017) "<a href="http://doi.org/10.1109/BIBM.2017.8217628">DiCE: Discovery of Conserved Noncoding Sequences Efficiently.</a>" <i>IEEE BIBM</i> Kansas City, MO, USA B529 (Conference Preceedings) doi: 10.1109/BIBM.2017.8217628

<div data-badge-type="2" data-doi="10.1093/nar/gkx853" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*  Ott A,* Liu S,* <b>Schnable JC</b>, Yeh CT, Wang C, Schnable PS. (2017) "<a href="https://academic.oup.com/nar/article/doi/10.1093/nar/gkx853/4210942/tGBS-genotypingbysequencing-enables-reliable?guestAccessKey=0ba723ae-c6b2-4d7f-841e-cd46b4bf68f9">Tunable Genotyping-By-Sequencing (tGBS) enables reliable genotyping of heterozygous loci.</a>" <i>Nucleic Acids Research</i> doi: 10.1093/nar/gkx853 <i>bioRxiv</i> doi: <a href="http://dx.doi.org/10.1101/100461">10.1101/100461</a>   

<div data-badge-type="2" data-doi="10.1186/s12864-017-4103-x" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*   <b>Lai X</b>, <b>Schnable JC</b>, Liao Z, Xu J, Zhang G, Li C, Hu E, Rong T, Xu Y, Lu Y. (2017) "<a href="https://doi.org/10.1186/s12864-017-4103-x">Genome-Wide Characterization of Non-Reference Transposable Elements Insertion Polymorphisms Reveals Genetic Diversity in Tropical and Temperate Maize.</a>" <i>BMC Genomics</i> doi: 10.1186/s12864-017-4103-x  

<div data-badge-type="2" data-doi="10.1534/genetics.117.300189" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*  Mei W, Boatwright L, Feng G, <b>Schnable JC</b>, Barbazuk WB. (2017) "<a href="https://doi.org/10.1534/genetics.117.300189">Evolutionarily conserved alternative splicing across monocots</a>." <i>Genetics</i> doi: 10.1534/genetics.117.300189 <i>bioRxiv</i> preprint doi: <a href="http://dx.doi.org/10.1101/120469">10.1101/120469</a><br>(Cover Article <a href="http://www.genetics.org/content/207/2">October 2017</a>)   

<div data-badge-type="2" data-doi="10.1105/tpc.17.00354" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*  <b>Zhang Y</b>, <b>Ngu DW</b>, <b>Carvalho D</b>, <b>Liang Z</b>, Qiu Y, Roston RL, <b>Schnable JC</b>. (2017) "<a href="http://dx.doi.org/10.1105/tpc.17.00354">Differentially regulated orthologs in sorghum and the subgenomes of maize.</a>" <i>The Plant Cell</i> doi: 10.1105/tpc.17.00354 <i>bioRxiv</i> preprint doi: <a href="http://dx.doi.org/10.1101/120303">10.1101/120303</a>  

<div data-badge-type="2" data-doi="10.3389/fpls.2017.01348" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*  Pandey P, Ge Y, Stoerger V, <b>Schnable JC</b>. (2017) "<a href="http://dx.doi.org/10.3389/fpls.2017.01348">High throughput in vivo analysis of plant leaf chemical properties using hyperspectral imaging</a>" <i>Frontiers in Plant Science</i> doi: 10.3389/fpls.2017.01348   

<div data-badge-type="2" data-doi="10.1016/j.molp.2017.05.010" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*  <b>Lai X</b>,* Behera S,* <b>Liang Z</b>, Lu Y, Deogun JS, <b>Schnable JC</b>. (2017) "<a href="http://dx.doi.org/10.1016/j.molp.2017.05.010">STAG-CNS: An order-aware conserved noncoding sequence discovery tool for arbitrary numbers of species.</a>" <i>Molecular Plant</i> doi: 10.1016/j.molp.2017.05.010 <i>bioRxiv</i> preprint doi: <a href="http://dx.doi.org/10.1101/120428">10.1101/120428</a>  

<div data-badge-type="2" data-doi="10.1016/j.molp.2017.05.009" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*  <b>Lai X</b>, <b>Schnable JC</b>. (2017) "<a href="https://doi.org/10.1016/j.molp.2017.05.009">Harnessing the potential of the tea tree genome.</a>" <i>Molecular Plant</i> doi: 10.1016/j.molp.2017.05.009  

<div data-badge-type="2" data-doi="10.3389/fpls.2017.00694" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*  Mei W, Liu S, <b>Schnable JC</b>, Yeh C, Springer NM, Schnable PS, Barbazuk WB. (2017) "<a href="https://doi.org/10.3389/fpls.2017.00694">A comprehensive analysis of alternative splicing in paleopolyploid maize.</a>" <i>Frontiers in Plant Science</i> doi: 10.3389/fpls.2017.00694  

**2016**

<div data-badge-type="2" data-doi="10.1093/bioinformatics/btw666" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* Joyce BL, Huag-Baltzell A, Davey S, Bomhoff M, <b>Schnable JC</b>, Lyons E. (2016) "<a href="http://dx.doi.org/10.1093/bioinformatics/btw666">FractBias: a graphical tool for assessing fractionation bias after whole genome duplications.</a>" <i>Bioinformatics</i> doi: 10.1093/bioinformatics/btw666

<div data-badge-type="2" data-doi="10.1186/s13059-016-1080-3" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* Studer AJ,* <b>Schnable JC</b>,* Weissmann S, Kolbe AR, McKain MR, Shao Y, Cousins AB, Kellogg EA, Brutnell TP. (2016) "<a href="http://dx.doi.org/10.1186/s13059-016-1080-3">The draft genome of Dichanthelium oligosanthes: A C3 panicoid grass species.</a>" <i>Genome Biology</i> doi: 10.1186/s13059-016-1080-3

<div data-badge-type="2" data-doi="10.1126/science.aag1125" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* Walley JW,* Sartor RC,* Shen Z, Schmitz RJ, Wu KJ, Urich MA, Nery JR, Smith LG, <b>Schnable JC</b>, Ecker JR, Briggs SP. (2016) "<a href="http://dx.doi.org/10.1126/science.aag1125">Integration of omic networks in a developmental atlas of maize.</a>" <i>Science</i> doi: 10.1126/science.aag1125

<div data-badge-type="2" data-doi="Pubs/Sruti_KDD_FEWS.pd" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* Chaudhury SD, Steorger V, Samal A, <b>Schnable JC</b>, <b>Liang Z</b>, Yu J. (2016) "<a href="Pubs/Sruti_KDD_FEWS.pdf">Automated vegetative stage phenotyping analysis of maize plants using visible light images.</a>" <i>KDD: Data Science for Food, Energy, and Water</i> San Francisco, CA, USA (Conference Proceedings)

<div data-badge-type="2" data-doi="10.1016/j.compag.2016.07.028" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* Ge Y, Bai G, Stoerger V, <b>Schnable JC</b>. (2016) "<a href="http://dx.doi.org/10.1016/j.compag.2016.07.028">Temporal dynamics of maize plant growth, water use, and plant water content using automated high throughput RGB and hyperspectral imaging.</a>" <i>Computers and Electronics in Agriculture</i> doi: 10.1016/j.compag.2016.07.028

<div data-badge-type="2" data-doi="10.1093/jxb/erw256" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* Huang P, Studer AJ, <b>Schnable JC</b>, Kellog EA, Brutnell TP. (2016) "<a href="http://dx.doi.org/10.1093/jxb/erw256">Cross species selection scans identify components of C4 photosynthesis in the grasses.</a>" <i>Journal of Experimental Botany</i> doi: 10.1093/jxb/erw256  

<div data-badge-type="2" data-doi="10.1371/journal.pone.0157942" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* <b>Liang Z</b>, <b>Schnable JC</b>. "<a href="http://dx.doi.org/10.1371/journal.pone.0157942">RNA-seq based analysis of population structure within the maize inbred B73.</a>" <i>PLoS One</i> doi: 10.1371/journal.pone.0157942 <i>bioRxiv</i> preprint doi: <a href="http://dx.doi.org/10.1101/043513">10.1101/043513</a>

<div data-badge-type="2" data-doi="10.1093/jxb/erw135" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* Liu X, Tang S, Jia G, <b>Schnable JC</b>, Su X, Tang C, Zhi H, Diao X. (2016) "<a href="http://dx.doi.org/10.1093/jxb/erw135">The C-terminal motif of SiAGO1b is required for the regulation of growth, development and stress responses in foxtail millet [Setaria italica (L.) P. Beauv]</a>" <i>Journal of Experimental Botany</i> doi: 10.1093/jxb/erw135  

<div data-badge-type="2" data-doi="10.1007/s11032-016-0460-4" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
*  Rajput SG, Santra DK, <b>Schnable JC</b>. (2016) "<a href="http://dx.doi.org/10.1007/s11032-016-0460-4">Mapping QTLs for morpho-agronomic traits in proso millet (<i>Panicum miliaceum L.</i>).</a>" <i>Molecular Breeding</i> doi: 10.1007/s11032-016-0460-4

<div data-badge-type="2" data-doi="10.1111/nph.13884" data-hide-no-mentions="true" data-hide-less-than="5" class="altmetric-embed" style="float:right;"></div>
* Cheng F, Sun C, Wu J, <b>Schnable JC</b>, Woodhouse MR, Liang J, Cai C, Freeling M, Wang X. (2016) "<a href="http://dx.doi.org/10.1111/nph.13884">Epigenetic regulation of subgenome dominance following whole genome triplication in Brassica rapa.</a>" <i>New Phytologist</i> doi: 10.1111/nph.13884

**2015**

* Chao S, Wu J, <b>Schnable JC</b>, Yang W, Cheng F, Wang X. (2015) "<a href="http://dx.doi.org/10.1093/gbe/evv206">Impacts of Whole Genome Triplication on MIRNA Evolution in Brassica rapa.</a>" <i>Genome Biology and Evolution</i> doi: 10.1093/gbe/evv206

* Tang H, Bamhoff MD, Briones E, <b>Schnable JC</b>, Lyons E. (2015) "<a href="http://dx.doi.org/10.1093/gbe/evv219">SynFind: compiling syntenic regions across any set of genomes on demand.</a>" <i>Genome Biology and Evolution</i> doi: 10.1093/gbe/evv219

* Washburn JD, <b>Schnable JC</b>, Davidse G, Pires JC. (2015) "<a href="http://dx.doi.org/10.3732/ajb.1500222">Phylogeny and photosynthesis of the grass tribe Paniceae.</a>" <i>American Journal of Botany</i> doi: 10.3732/ajb.1500222

* Jia G, Liu X, <b>Schnable JC</b>, Niu Z, Wang C, Li Y, Wang Sh, Wang Su, Liu J, Gou E, Diao X. (2015) "<a href="http://dx.doi.org/10.1371/journal.pone.0125688">Microsatellite Variations of Elite Setaria Varieties Released during Last Six Decades in China.</a>" <i>PLoS One</i> doi:10.1371/journal.pone.0125688

* Tang H, Zhang X, <b>Miao C</b>, Zhang J, Ming R, <b>Schnable JC</b>, Schnable PS, Lyons E, Lu Jianguo. (2015) "<a href="http://dx.doi.org/10.1186/s13059-014-0573-1">ALLMAPS: robust scaffold ordering based on multiple maps.</a>" <i>Genome Biology</i> doi:10.1186/s13059-014-0573-1

* <b>Schnable JC</b>. (2015) "<a href="http://dx.doi.org/10.1146/annurev-arplant-043014-115604">Genome evolution in maize: from genomes back to genes.</a>" <i>Annual Review of Plant Biology</i> doi:10.1146/annurev-arplant-043014-115604

**2014**

* Paschold A, Larson NB, Marcon C, <b>Schnable JC</b>, Yeh C, Lanz C, Nettleton D, Piepho H, Schnable PS, Hochholdinger F. (2014) "<a href="http://dx.doi.org/10.1105/tpc.114.130948">Nonsyntenic Genes Drive Highly Dynamic Complementation of Gene Expression in Maize Hybrids</a>" <i>Plant Cell</i> doi:10.1105/tpc.114.130948

* Almeida AMR, Yockteng R, <b>Schnable JC</b>, Alvarez-Buylla ER, Freeling M, Specht CD. (2014) "<a href="http://dx.doi.org/10.1038/srep06194">Co-option of the polarity gene network shapes filament morphology in angiosperms</a>" <i>Scientific Reports</i> doi: 10.1038/srep06194

* Qie L, Jia G, Zhang W, <b>Schnable JC</b>, Shang Z, Li W, Liu B, Li M, Chai, Y, Zhi H, Diao X. (2014) "<a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0101868">Mapping of Quantitative Trait Locus (QTLs) that Contribute to Germination and Early Seedling Drought Tolerance in the Interspecific Cross <i>Setaria italica</i> x <i>Setaria viridis</i>.</a>" <i>PLoS One</i> doi: 10.1371/journal.pone.0101868

* Diao X, <b>Schnable JC</b>, Bennetzen JL, Li J. (2014) "<a href="http://engineering.cae.cn/fase/EN/10.15302/J-FASE-2014011">Initiation of Setaria as a model plant.</a>" <i>Frontiers of Agricultural Science and Engineering</i> doi: 10.15302/J-FASE-2014011

* Martin JA, Johnson NV, Gross SM, <b>Schnable JC</b>, Meng X, Wang M, Coleman-Derr D, Lindquist E, Wei C, Kaeppler S, Chen F, Wang Z. (2014) "<a href="http://www.nature.com/srep/2014/140331/srep04519/full/srep04519.html">A near complete snapshot of the Zea mays seedling transcriptome revealed from ultra-deep sequencing.</a>" <i>Scientific Reports</i> doi: 10.1038/srep04519

**2013**

* Garsmeur O,* <b>Schnable JC,*</b> Almeida A, Jourda C, D'Hont A, Freeling M (2013) "<a href="http://dx.doi.org/10.1093/molbev/mst230">Two Evolutionarily Distinct Classes of Paleopolyploidy.</a>" <i>Molecular Biology and Evolution</i> doi: 10.1093/molbev/mst230

* Turco G, <b>Schnable JC</b>, Bedersen B, Freeling M. (2013) "<a href="http://dx.doi.org/10.3389/fpls.2013.00170">Automated conserved noncoding sequence (CNS) discovery reveals differences in gene content and promoter evolution among the grasses.</a>" <i>Frontiers in Plant Sciences</i> doi: 10.3389/fpls.2013.00170

**2012**

* <b>Schnable JC</b>, Wang X, Pires JC, Freeling M. (2012) "<a href="http://dx.doi.org/10.3389/fpls.2012.00094">Escape from preferential retention following repeated whole genome duplication in plants.</a>" <i>Frontiers in Plant Science</i> doi: 10.3389/fpls.2012.00094

* Freeling M, Woodhouse MR, Subramaniam S, Turco G, Lisch D, <b>Schnable JC</b>. (2012) "<a href="http://dx.doi.org/10.1016/j.pbi.2012.01.015">Fractionation mutagenesis and similar consequences of mechanisms removing dispensable or less-expressed DNA in plants</a>" <i>Current Opinion in Plant Biology</i> doi: 10.1016/j.pbi.2012.01.015

* Tang H, Woodhouse MR, Cheng F, <b>Schnable JC</b>, Pedersen BS, Conant GC, Wang X, Freeling M, Pires JC. (2012) "<a href="http://dx.doi.org/10.1534/genetics.111.137349">Altered patterns of fractionation and exon deletions in Brassica rapa support a two-step model of paleohexaploidy.</a>" <i>Genetics</i> doi: 10.1534/genetics.111.137349

* <b>Schnable JC</b>, Freeling M, Lyons E. (2012) "<a href="http://dx.doi.org/10.1093/gbe/evs009">Genome-wide analysis of syntenic gene deletion in the grasses</a>" <i>Genome Biology and Evolution</i> doi: 10.1093/gbe/evs009

* Zhang W, Wu Y, <b>Schnable JC,</b> Zeng Z, Freeling M, Crawford GE, and Jiang J. (2012) "<a href="http://dx.doi.org/10.1101/gr.131342.111">High-resolution mapping of open chromatin in the rice genome.</a>" <i>Genome Research</i> doi: 10.1101/gr.131342.111

**2011**

* Eichten SR,* Swanson-Wagner RA,* <b>Schnable JC,</b> Waters AJ, Hermanson PJ, Liu S, Yeh C, Jia Y, Gendler K, Freeling M, Schnable PS, Vaughn MW, Springer NM. (2011) "<a href="http://dx.doi.org/10.1371/journal.pgen.1002372">Heritable epigenetic variation among maize inbreds.</a>" <i>PLoS Genetics</i> doi: 10.1371/journal.pgen.1002372

* <b>Schnable JC</b>, Lyons E. (2011) "<a href="http://www.schnablelab.org/Pubs/56_183.pdf">Comparative genomics with maize and other grasses: from genes to genomes</a>" <i>Maydica</i> (Link is directly to PDF)

* Tang H, Lyons E, Pedersen B, <b>Schnable JC</b>, Paterson AH, Freeling M. (2011) "<a href="http://www.biomedcentral.com/1471-2105/12/102/abstract">Screening synteny blocks in pairwise genome comparisons through integer programming.</a>" <i>BMC Bioinformatics</i> doi: 10.1186/1471-2105-12-102

* <b>Schnable JC</b>, Pedersen BS, Subramaniam S, Freeling M. (2011) "<a href="http://www.frontiersin.org/plant_genetics_and_genomics/10.3389/fpls.2011.00002/abstract">Dose-sensitivity, conserved noncoding sequences and duplicate gene retention through multiple tetraploidies in the grasses.</a>" <i>Frontiers in Plant Science</i> doi: 10.3389/fpls.2011.00002
<br><i>(<a href="http://www.frontiersin.org/plant%20genetics%20and%20genomics/10.3389/fpls.2011.00064/full">Commentary on this article</a> by Birchler and Veitia, also published in Frontiers in Plant Science)</i>

* <b>Schnable JC</b>, Freeling M. (2011) "<a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0017855">Genes identifed by visible mutant phenotypes show increased bias towards one of two maize subgenomes.</a>" <i>PLoS One</i> doi: 10.1371/journal.pone.0017855

* <b>Schnable JC</b>, Springer NM, Freeling M. (2011) "<a href="http://www.pnas.org/content/108/10/4069.abstract">Differentiation of the maize subgenomes by genome dominance and both ancient and ongoing gene loss.</a>" <i>PNAS</i> doi: 10.1073/pnas.1101368108 <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3053962/">Read for free at PubMed Central</a>
            <br><i>(Selected as an Editor's Choice by MaizeGDB May 2011)</i>

**2010**

* Woodhouse MR,* <b>Schnable JC,*</b> Pedersen BS, Lyons E, Lisch D, Subramaniam S, Freeling M. (2010) "<a href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000409">Following tetraploidy in maize, a short deletion mechanism removed genes preferentially from one of the two homeologs.</a>" <i>PLoS Biology</i> doi:10.1371/journal.pbio.1000409
            <br><i>(Selected as an <a href="http://maizeresearch.org/2010/08/tetraploidy-in-maize/">Editor's Choice by MaizeGDB August 2010</a>; an image related to this manuscript was used for <a href="http://www.plosbiology.org/article/browseIssue.action?issue=info%3Adoi%2F10.1371%2Fissue.pbio.v08.i06#top">the journal cover</a>; "recommended" by Faculty of 1000)</i>

* The International Brachypodium Initiative. (2010) "<a href="http://dx.doi.org/10.1038/nature08747">Genome sequencing and analysis of the model grass Brachypodium distachyon.</a>" <i>Nature</i> doi: 10.1038/nature08747 _*Authors Contributed Equally_
