---
layout: page
title: Schnable Lab Research Interests
---

Research in the Schnable Lab@UNL in an interdisciplinary business. Our projects involving collaborations with engineers, computer scientists, statisticians, and food scientists. Here are three main clusters:
<ul>
<li><a href="#Comparative">Comparative functional genomics</a></li>
<li><a href="#Phenotyping">High throughput phenotyping</a></li>
<li><a href="#QuantGen">Quantitative genetics</a></li>
<li><a href="#Orphan">Miscellaneous Other Cool Science</a></li>
</ul>

<a id="Comparative">

**Comparative Functional Genomics**

Research in this area focuses on utilizing cross species comparisons to separate out the functional and functionless portions of plant genomes and then prediction the functions of the apparently functional bits. 


**Recent Lab Publications on Plant Comparative Omics**

* **Schnable JC** (2019) "Genes and Gene Models, an Important Distinction." *New Phytologist* *(In Press)*
* **Yan L**, **Raju SKK**, **Lai X**, **Zhang Y**, **Dai X**, Rodriguez O, Mahboub S, Roston RL, **Schnable JC** (2019) "[Parallel natural selection in the cold-adapted crop-wild relative *Tripsacum dactyloides* and artificial selection in temperate adapted maize.](https://doi.org/10.1111/tpj.14376)" *The Plant Journal* doi: 10.1111/tpj.14376 *bioRxiv* doi: 10.1101/187575
* **Raju SKK**, Barnes A, **Schnable JC**, Roston RL. (2018) "[Low-temperature tolerance in land plants: Are transcript and membrane responses conserved?](https://doi.org/10.1016/j.plantsci.2018.08.002)" *Plant Science* doi: 10.1016/j.plantsci.2018.08.002
* **Liang Z**, **Schnable JC**. (2018) "<a href="https://doi.org/10.1016/j.molp.2017.12.010">Functional Divergence Between Subgenomes and Gene Pairs After Whole Genome Duplications.</a>" <i>Molecular Plant</i> doi: 10.1016/j.molp.2017.12.010
* <b>Lai X</b>, <b>Yan L</b>, Lu Y, <b>Schnable JC</b>. (2018) "<a href="https://doi.org/10.1111/tpj.13806">Largely unlinked gene sets targeted by selection for domestication syndrome phenotypes in maize and sorghum.</a>" <i>The Plant Journal</i> doi: 10.1111/tpj.13806  <i>bioRxiv</i> doi: <a href="https://doi.org/10.1101/184424">10.1101/184424</a>
*  Mei W, Boatwright L, Feng G, <b>Schnable JC</b>, Barbazuk WB. (2017) "<a href="https://doi.org/10.1534/genetics.117.300189">Evolutionarily conserved alternative splicing across monocots</a>." <i>Genetics</i> doi: 10.1534/genetics.117.300189 <i>bioRxiv</i> preprint doi: <a href="http://dx.doi.org/10.1101/120469">10.1101/120469</a><br>(Cover Article <a href="http://www.genetics.org/content/207/2">October 2017</a>)   
*  <b>Zhang Y</b>, <b>Ngu DW</b>, <b>Carvalho D</b>, <b>Liang Z</b>, Qiu Y, Roston RL, <b>Schnable JC</b>. (2017) "<a href="http://dx.doi.org/10.1105/tpc.17.00354">Differentially regulated orthologs in sorghum and the subgenomes of maize.</a>" <i>The Plant Cell</i> doi: 10.1105/tpc.17.00354 <i>bioRxiv</i> preprint doi: <a href="http://dx.doi.org/10.1101/120303">10.1101/120303</a>  
*  <b>Lai X</b>,* Behera S,* <b>Liang Z</b>, Lu Y, Deogun JS, <b>Schnable JC</b>. (2017) "<a href="http://dx.doi.org/10.1016/j.molp.2017.05.010">STAG-CNS: An order-aware conserved noncoding sequence discovery tool for arbitrary numbers of species.</a>" <i>Molecular Plant</i> doi: 10.1016/j.molp.2017.05.010 <i>bioRxiv</i> preprint doi: <a href="http://dx.doi.org/10.1101/120428">10.1101/120428</a>  
* Walley JW,* Sartor RC,* Shen Z, Schmitz RJ, Wu KJ, Urich MA, Nery JR, Smith LG, <b>Schnable JC</b>, Ecker JR, Briggs SP. (2016) "<a href="http://dx.doi.org/10.1126/science.aag1125">Integration of omic networks in a developmental atlas of maize.</a>" <i>Science</i> doi: 10.1126/science.aag1125
* <b>Schnable JC</b>. (2015) "<a href="http://dx.doi.org/10.1146/annurev-arplant-043014-115604">Genome evolution in maize: from genomes back to genes.</a>" <i>Annual Review of Plant Biology</i> doi: 10.1146/annurev-arplant-043014-115604

**Funding Supporting Efforts in This Cluster**

* NSF [RoL: FELS: EAGER: Genetic Constraints on the Increase of Organismal Complexity Over Time.](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1838307)
* USDA-NIFA [Identifying mechanisms conferring low temperature tolerance in maize, sorghum, and frost tolerant relatives.](https://portal.nifa.usda.gov/web/crisprojectpages/1008702-identifying-mechanisms-conferring-low-temperature-tolerance-in-maize-sorghum-and-frost-tolerant-relatives.html)

<a id="Phenotying"></a>
<br>
**High Throughoupt  Phenotyping**
<br>
This area focuses on developing and deploying new algorithms, tools, and datasets for high throughput plant phenotyping. On the development side we collaborate closely with engineers and statisticians both here at UNL and at other univerisities around the world.

**Recent Lab Publications on High Throughput Phenotyping**
* Ge Y, Atefi A, Zhang H, **Miao C**, Ramamurthy RK, **Sigmon B**, Yang J, **Schnable JC** (2019) "High-throughput analysis of leaf physiological and chemical traits with VIS-NIR-SWIR spectroscopy: A case study with a maize diversity panel." *Plant Methods* *(Accepted)*
* Atefi A, Ge Y,  Pitla S, **Schnable JC** (2019) [*In vivo* human-like robotic phenotyping of leaf traits in maize and sorghum.](https://doi.org/10.1016/j.compag.2019.104854) *Computers and Electronics in Agriculture* doi: 10.1016/j.compag.2019.104854
* Bai G, Ge Y, Scoby D, Leavit B, Irmak S, Graef G, **Schnable JC**, Awada T. (2019) "[NU-Spidercam: A large-scale, cable-driven, integrated sensing and robotic system for precision phenotyping, remote sensing, and agronomic research.](https://authors.elsevier.com/sd/article/S0168169918314170)" *Computers and Electronics in Agriculture* doi: 10.1016/j.compag.2019.03.009
* Alkhalifah N, Campbell DA, Falcon CM, ... **Schnable JC** (31 of 44 authors) ... Spalding EP, Edwards J, Lawrence-Dill CJ. (2018) "[Maize Genomes to Fields: 2014 and 2015 field season genotype, phenotype, environment, and inbred ear image datasets.](https://doi.org/10.1186/s13104-018-3508-1)" <i>BMC Research Notes</i> doi: https://doi.org/10.1186/s13104-018-3508-1
* Xu Y, Qiu Y, **Schnable JC**. (2018) "[Functional Modeling of Plant Growth Dynamics.](https://dl.sciencesocieties.org/publications/tppj/abstracts/1/1/170007)" <i>The Plant Phenome</i> doi: 10.2135/tppj2017.09.0007 <i>bioRxiv</i> doi: <a href="https://doi.org/10.1101/190967">10.1101/190967</a>
*  <b>Liang Z</b>, Pandey P, Stoerger V, Xu Y, Qiu Y, Ge Y, <b>Schnable JC</b>. (2017) "<a href="https://academic.oup.com/gigascience/advance-article/doi/10.1093/gigascience/gix117/4656251?guestAccessKey=71c1c32f-78fd-42c0-99a3-38e6c4fd8100">Conventional and hyperspectral time-series imaging of maize lines widely used in field trials.</a>" <i>GigaScience</i> doi: 10.1093/gigascience/gix117 <i>bioRxiv</i> doi: <a href="https://doi.org/10.1101/169045">10.1101/169045</a>
*  Gage J, Jarquin D, Romay M, ... <b>Schnable JC</b> (29th of 40 authors) .. Yu J, de Leon N. (2017) "<a href="https://doi.org/10.1038/s41467-017-01450-2">The effect of artificial selection on phenotypic plasticity in maize.</a>" <i>Nature Communications</i> doi: 10.1038/s41467-017-01450-2  
*  Pandey P, Ge Y, Stoerger V, <b>Schnable JC</b>. (2017) "<a href="http://dx.doi.org/10.3389/fpls.2017.01348">High throughput in vivo analysis of plant leaf chemical properties using hyperspectral imaging</a>" <i>Frontiers in Plant Science</i> doi: 10.3389/fpls.2017.01348  
* Ge Y, Bai G, Stoerger V, <b>Schnable JC</b>. (2016) "<a href="http://dx.doi.org/10.1016/j.compag.2016.07.028">Temporal dynamics of maize plant growth, water use, and plant water content using automated high throughput RGB and hyperspectral imaging.</a>" <i>Computers and Electronics in Agriculture</i> doi: 10.1016/j.compag.2016.07.028 

**Funding Supporting Efforts in This Cluster**

* NSF [BTT EAGER: A wearable plant sensor for real-time monitoring of sap flow and stem diameter to accelerate breeding for water use efficiency.](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1844563) 
* ARPA-E: In-plant and in-soil microsensors enabled high-throughput phenotyping of root nitrogen uptake and nitrogen use efficiency."
* ARPA-E: Low cost wireless chemical sensor networks."
* USDA/NSF Joint Program [PAPM EAGER: Transitioning to the next generation plant phenotyping robots.](https://reeis.usda.gov/web/crisprojectpages/1011763-papm-eager-transitioning-to-the-next-generation-plant-phenotyping-robots.html)
* North Central Sun Grants: High through put phenotyping to accelerate biomass sorghum improvement.
* Nebraska Corn Board: Genomes to Fields (G2F) - Predicting Final Yield Performance in Variable Environments.
* Wheat Innovation Foundation: A Low-Cost, High-Throughput Cold Stress Perception Assay for Sorghum Breeding.


<a id="QuantGen" />

**Quantitative Genetics**

At its most basic, this area ensures that graduates of the Schnable lab are familiar with modern algorithms to perform basic quantitative genetic tasks including GWAS and genomic selection. However, more advanced work in this area involves developing and applying new statistical approaches that either leverage either the unique features of new phenotypic datasets (for example time-series data from whole mapping populations collected using HTP technologies) or sharing data across related species. 

**Recent Lab Publications on Quantitative Genetics**

* **Liang Z**, Qiu Y, **Schnable JC** (2018) "[Distinct characteristics of genes associated with phenome-wide variation in maize (*Zea mays*)](https://doi.org/10.1101/534503)" *bioRxiv* doi: 10.1101/534503
* **Miao C**, **Yang J**, **Schnable JC** (2018) "[Optimizing the identification of causal variants across varying genetic architectures in crops.](https://doi.org/10.1111/pbi.13023)" *Plant Biotechnology Journal* doi: 10.1111/pbi.13023 <i>bioRxiv</i> doi: [10.1101/310391](https://doi.org/10.1101/310391)
* <b>Liang Z</b>, Gupta SK, Yeh CT, <b>Zhang Y</b>, <b>Ngu DW</b>, Kumar R, Patil HT, Mungra KD, Yadav DV, Rathore A, Srivastava RK, Gupkta R, <b>Yang J</b>, Varshney RK, Schnable PS, <b>Schnable JC</b>. (2018) "[Phenotypic data from inbred parents can improve genomic prediction in pearl millet hybrids.](https://doi.org/10.1534/g3.118.200242)" <i>G3: Genes Genomes Genetics</i> doi: 10.1534/g3.118.200242
* Li L, Li X, Li L, **Schnable JC**, Gu R, J Wang (2019) "[QTL identification and epistatic effect analysis of seed size- and weight-related traits in *Zea mays* L.](https://doi.org/10.1007/s11032-019-0981-8)" *Molecular Breeding* doi: 10.1007/s11032-019-0981-8
* **Miao C**, Fang J, Li D, Liang P, Zhang X, **Yang J**, **Schnable JC**, Tang H. (2018) "[Genotype-Corrector: improved genotype calls for genetic mapping.](https://doi.org/10.1038/s41598-018-28294-0)" <i>Scientific Reports</i> doi: 10.1038/s41598-018-28294-0
*  Ott A,* Liu S,* <b>Schnable JC</b>, Yeh CT, Wang C, Schnable PS. (2017) "<a href="https://academic.oup.com/nar/article/doi/10.1093/nar/gkx853/4210942/tGBS-genotypingbysequencing-enables-reliable?guestAccessKey=0ba723ae-c6b2-4d7f-841e-cd46b4bf68f9">Tunable Genotyping-By-Sequencing (tGBS) enables reliable genotyping of heterozygous loci.</a>" <i>Nucleic Acids Research</i> doi: 10.1093/nar/gkx853 <i>bioRxiv</i> doi: <a href="http://dx.doi.org/10.1101/100461">10.1101/100461</a>   
* <b>Liang Z</b>, <b>Schnable JC</b>. "<a href="http://dx.doi.org/10.1371/journal.pone.0157942">RNA-seq based analysis of population structure within the maize inbred B73.</a>" <i>PLoS One</i> doi: 10.1371/journal.pone.0157942 <i>bioRxiv</i> preprint doi: <a href="http://dx.doi.org/10.1101/043513">10.1101/043513</a>
*  Rajput SG, Santra DK, <b>Schnable JC</b>. (2016) "<a href="http://dx.doi.org/10.1007/s11032-016-0460-4">Mapping QTLs for morpho-agronomic traits in proso millet (<i>Panicum miliaceum L.</i>).</a>" <i>Molecular Breeding</i> doi: 10.1007/s11032-016-0460-4

**Funding Supporting Efforts in This Cluster**

* NSF [RII Track-2 FEC: Functional analysis of nitrogen responsive networks in Sorghum.](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1826781&HistoricalAwards=false)
* FFAR [Crops in silico: Increasing crop production by connecting models from the microscale to the macroscale.](http://www.ncsa.illinois.edu/news/story/crops_in_silico_project_awarded_5_million)
* NSF [Center for Root and Rhizobiome Innovation.](https://nsf.gov/awardsearch/showAward?AWD_ID=1557417)
* ICRISAT: Application of tGBS And Genomic Selection to a Hybrid Pearl Millet Breeding Program.

